d3a_interface/sim_results/simulation_assets_info.py

```diff
@@ -21,50 +24,52 @@ def merge_results_to_global(market_results: Dict, global_results: Dict, slot_lis
             "merge not supported.")
 
     def update(self, area_result_dict: Dict, core_stats: Dict, current_market_slot: str):
+        # total_energy_demanded_wh is already accumulated value and should only be replaced
         updated_results_dict = {
-            'number_of_load_type': 0,
-            'total_energy_demand_kwh': self.assets_info['total_energy_demand_kwh'],
-            'number_of_pv_type': 0,
-            'total_energy_generated_kwh': self.assets_info['total_energy_generated_kwh']
+            "number_of_load_type": 0,
+            "total_energy_demand_kwh": 0,
```

Comment:
Could you explain why/how this change fixes the reported bug?

Moreover, in the ticket I read:
> Secondly, my PV has sold 12.29 kWh, so I would expect its PV, Total Generated to be 12.29 kWh instead of 0 kWh

so I would have expected also `total_energy_generated_kwh` to be modified, but that's not the case. That's why I'd like to know more about the reason why the solution works :)

Reply:
I tried to explain it in my comment in Line 27 : `total_energy_demanded_wh is already accumulated value and should only be replaced` , whereas `total_energy_generated_kwh` only applies to the last market slot for the PVs. 
This mismatch is not ideal and if you agree I could add a refactoring story where we homogenise the sent values on the d3a side.

Reply:
Ahh ok, so `total_energy_demanded_wh` is already accumulated (somewhere else in the code), so we just need to set its values as we receive it. Yes, if possible I would vote for creating a ticket for homogenizing it!

Reply:
Done https://gridsingularity.atlassian.net/browse/D3ASIM-3546